src/sssom/util.py

```diff
@@ -1189,6 +1189,8 @@ def filter_out_prefixes(
     rows = []
     selection = all if require_all_prefixes else any
 
+    df[PREDICATE_MODIFIER] = "" if PREDICATE_MODIFIER not in df.columns else df[PREDICATE_MODIFIER]
```

Comment:
Why is this change in this PR?

Reply:
I took that off in another PR. It should not be present at all.

Reply:
Rerunning the workflow fixed it automatically.